src/help.h

```diff
@@ -1322,7 +1322,17 @@ struct commandHelp {
     "destination numkeys key [key ...] [WEIGHTS weight] [AGGREGATE SUM|MIN|MAX]",
     "Add multiple sorted sets and store the resulting sorted set in a new key",
     4,
-    "2.0.0" }
+    "2.0.0" },
+    { "ZDIFF",
+    "numkeys key [key ...] [WITHSCORES]",
+    "Subtract multiple sorted sets from the first set and return the resulting set elements",
+    4,
+    "6.2.0" },
+    { "ZDIFFSTORE",
+    "destination numkeys key [key ...]",
+    "Subtract multiple sorted sets from the first set and store the resulting sorted set in a new key",
+    4,
+    "6.2.0" }
```

Comment:
> i see t_set.c saying this:
> 
> ```c
>             /* With algorithm 1 it is better to order the sets to subtract
>              * by decreasing size, so that we are more likely to find
>              * duplicated elements ASAP. */
>             qsort(sets+1,setnum-1,sizeof(robj*),
>                 qsortCompareSetsByRevCardinality);
> ```
> 
> maybe we need to do the same in zset?
> 

Yes, forgot about that, I'll add it here.

Reply:
> this file is auto generated (from commands.json in redis-doc repo).
> we re-generate it when we create an actual release, so the copy in unstable is rather outdated.
> there's no need to add these, but on the other hand they don't cause any harm (will get overwritten)

Totally forgot about that, I think I'll remove it just to avoid modifying this file for nothing.

Reply:
Done, both the qsort and reverting the "help.h" file.